07.high-fidelity-commerce/01.basics/01.hfc/docs.md

```diff
@@ -10,11 +10,9 @@ taxonomy:
 
 High Fidelity Coins is a cryptocurrency that is empowered by blockchain technology.
 
-We are currently giving out the currency for anyone interested in participating in the closed beta for High Fidelity Commerce. If you want to [get your inital HFC grant](/high-fidelity-commerce/basics/hfc/get-hfc), you first need to [set up your Wallet](https://docs.highfidelity.com/high-fidelity-commerce/basics/hfc/wallet-setup).
+We are currently giving out the currency for anyone interested in participating in the closed beta for High Fidelity Commerce. These coins are to be used as currency for any commerce transactions in the Marketplace. Since we are using blockchain technology, all transactions with HFCs will be publically recorded and stored. 
```

Comment:
Looks like "publicly" is more widely accepted than "publically". Does it matter?
http://www.macmillandictionaryblog.com/is-there-a-case-for-publically-part-1

Reply:
I agree with you...this was existing documentation. I'll go ahead and change it.